usb/webusb.go

```diff
@@ -197,6 +197,11 @@ func (b *WebUSB) connect(dev lowlevel.Device) (*WUD, error) {
 }
 
 func (b *WebUSB) match(dev lowlevel.Device) bool {
+	if isWindows {
```

Comment:
Maybe it's easier to just check whether `runtime.GOOS` is `windows/386` or `windows/amd64` rather than introducing 2 new files. But this is just a nitpick.

Reply:
I have read somewhere that it is not so dependable and compile time is better

Reply:
... but you are probably right. I will edit the commit

Reply:
OK, I will edit the commit, you are probably right